react/Alert/Alert.less

```diff
@@ -51,7 +51,8 @@
     vertical-align: top;
   }
 
-  svg { // stylelint-disable-line selector-max-type
+  svg {
+    // stylelint-disable-line selector-max-type
```

Comment:
This looks wrong, probably dont need the ignore anymore if its not breaking the build, not immediately obvious how that would trigger the max selector rule

Reply:
Linting passed when removed. Must have been leftover when something changed.